src/containers/tutor-home-page/subjects-step/SubjectsStep.jsx

```diff
@@ -107,7 +114,8 @@ const SubjectsStep = ({ stepLabel, btnsBox }) => {
             disabled={!subjects.category}
             labelField='name'
             onChange={onChangeSubject}
-            service={getSubjectsNames}
+            onFocus={handleSubjectFetch}
+            service={subjectFetched && getSubjectsNames}
```

Comment:
i think here u have ts error

Reply:
> maybe you need to modify this autocomplete, and provide just service in it as it expects

I have no choice but to provide onFocus prop or get item data when changing category, which will not satisfy the condition of the task. So i think, there are no other options to track user onFocus event